CMakeLists.txt

```diff
@@ -13,6 +13,11 @@ set (SOVERSION "10")
 
 include(GNUInstallDirs)
 include(GeneratePkgConfig)
+include(CMakeDependentOption)
+
+# Boost
+find_public_dependency(Boost REQUIRED COMPONENTS system)
```

Comment:
Why don't you modify https://github.com/arvidn/libtorrent/blob/master/CMakeLists.txt#L681?

Reply:
Never mind, pardon.